src/static/current-state/components/tabs/index.tsx

```diff
@@ -100,7 +100,7 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
     }
 
     this.state = {
-      selected: isNaN(selected) ? 1 : selected,
+      selected: !selected || isNaN(selected) ? 1 : selected,
```

Comment:
Have you checked the logic?

Reply:
I added this because isNaN on an empty string returns false, but seems like it's unnecessary since parseInt on an empty string is NaN.

Reply:
yeap